tck/src/main/java/org/reactivestreams/tck/SubscriberWhiteboxVerification.java

```diff
@@ -266,8 +266,9 @@ public void required_spec208_mustBePreparedToReceiveOnNextSignalsAfterHavingCall
       @Override
       public void run(WhiteboxTestStage stage) throws InterruptedException {
         stage.puppet().triggerRequest(1);
-        stage.puppet().signalCancel();
         stage.expectRequest();
+        stage.puppet().signalCancel();
+        stage.expectCancelling();
```

Comment:
Now we expect the request before signalling cancel, preventing the coalescing of the request and cancel, and by expecting cancelling before signalling next, we ensure that signalling next does test what this test says it tests.

Reply:
Sounds reasonable and less prone to race conditions.

Reply:
+1. the behavior you described sounds like an improvement.